<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedgerMainWindow</class>
 <widget class="QMainWindow" name="LedgerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Ledger</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="MainTabs">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="TabMain">
       <attribute name="title">
        <string>Balance</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="BalanceOperationsSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="BalanceBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Balances</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="BalanceConfigFrame">
              <property name="minimumSize">
               <size>
                <width>408</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>44</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="leftMargin">
                <number>2</number>
               </property>
               <property name="topMargin">
                <number>2</number>
               </property>
               <property name="rightMargin">
                <number>2</number>
               </property>
               <property name="bottomMargin">
                <number>2</number>
               </property>
               <item>
                <widget class="QDateEdit" name="BalanceDate"/>
               </item>
               <item>
                <widget class="QLabel" name="CurrencyLbl">
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Sum Currency:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="CurrencyCombo"/>
               </item>
               <item>
                <widget class="QCheckBox" name="ShowInactiveCheckBox">
                 <property name="text">
                  <string>Show &amp;Inactive</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="BalancesTableView">
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DotLine</enum>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="OperationsBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>4</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="title">
            <string>Operations</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSplitter" name="OperationsDetailsSplitter">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTableView" name="OperationsTableView"/>
              <widget class="QFrame" name="OperationDetails">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>250</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTabWidget" name="OperationsTabs">
                  <property name="tabPosition">
                   <enum>QTabWidget::South</enum>
                  </property>
                  <property name="tabShape">
                   <enum>QTabWidget::Triangular</enum>
                  </property>
                  <property name="currentIndex">
                   <number>0</number>
                  </property>
                  <widget class="QWidget" name="ActionDetailsTab">
                   <attribute name="title">
                    <string>Income / Spending</string>
                   </attribute>
                  </widget>
                  <widget class="QWidget" name="TradeDetailsTab">
                   <attribute name="title">
                    <string>Buy / Sell</string>
                   </attribute>
                  </widget>
                  <widget class="QWidget" name="DividendDetailsTab">
                   <attribute name="title">
                    <string>Dividend</string>
                   </attribute>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="TabTransactions">
       <attribute name="title">
        <string>Transactions</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_DAta">
    <property name="title">
     <string>&amp;Data</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Load_quotes"/>
    <addaction name="actionLoad_Statement"/>
    <addaction name="separator"/>
    <addaction name="action_Import"/>
    <addaction name="action_Re_build_Ledger"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_DAta"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Re_build_Ledger">
   <property name="text">
    <string>&amp;Re-build Ledger</string>
   </property>
  </action>
  <action name="action_Import">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="action_Load_quotes">
   <property name="text">
    <string>Load &amp;quotes</string>
   </property>
  </action>
  <action name="actionLoad_Statement">
   <property name="text">
    <string>Load &amp;Statement...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
